gradle/libs.versions.toml

```diff
@@ -1,25 +1,25 @@
 [versions]
-android-gradle-plugin = "8.9.2"
+android-gradle-plugin = "8.10.0"
 activity-compose = "1.10.1"
 constraintlayout-compose = "1.1.1"
 compose-material = "1.7.8"
-compose = "2025.04.00"
+compose = "2025.05.00"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/50f493ab1421e4fbba96bc4ef10078e5d9a0205d/gradle/libs.versions.toml#L6"></span>A newer version of androidx.compose:compose-bom than 2025.05.00 is available: 2025.05.01</td>
    </tr>
  </tbody>
</table>